wpengine/faustjs
# feat (BH-864): Add a category template to the getting-started example

Adds a `category.tsx` template with a custom query, so that category archives such as `category/uncategorized` load category-specific posts instead of latest posts. For [BH-864](https://wpengine.atlassian.net/browse/BH-864).

Thanks to @mindctrl for working on this with me.

## To test
1. `npm run bootstrap && npm run dev` from the repo root.
2. Visit http://localhost:3000/category/uncategorized or http://localhost:3000/category/your-category-slug.

You should see the most recent 10 posts in the named category.

Pagination will be added in a separate PR.
